.github/workflows/r.yml

```diff
@@ -40,6 +40,10 @@ jobs:
     env:
       SPARK_VERSION: ${{ matrix.spark }}
       SCALA_VERSION: ${{ matrix.scala }}
+      # Ensure the temporary auth token for this workflow, instead of the
+      # bundled GitHub PAT from the `remotes` package is used for
+      # `remotes::install_github()`
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
OK. Looks like this definitely fixes all of the R package installation failure(s) from existing R builds.